gcalbot/gcalbot/event.go

```diff
@@ -99,3 +100,38 @@ func FormatEvent(
 	return fmt.Sprintf(message,
 		url, what, when, where, conferenceData, organizer, calendarSummary, description), nil
 }
+
+func FormatEventSchedule(
+	events []*calendar.Event,
+	timezone *time.Location,
+	format24HourTime bool,
+) (schedule string, err error) {
+	// don't know the length of the array since we skip all-day events
+	sort.Slice(events, func(i, j int) bool {
+		startLeft, _, _, _ := ParseTime(events[i].Start, events[i].End)
+		startRight, _, _, _ := ParseTime(events[j].Start, events[j].End)
+		return startLeft.Before(startRight)
+	})
+	var formattedEvents []string
+	for _, event := range events {
+		start, end, isAllDay, err := ParseTime(event.Start, event.End)
+		if err != nil {
+			return "", err
+		}
+		if isAllDay {
+			// TODO(marcel): support all day events
+			continue
+		}
+		start = start.In(timezone)
+		end = end.In(timezone)
+
+		if format24HourTime {
```

Comment:
can you just call `start.Format`/`end.Format` in these conditionals so the event formatting isn't duplicated?